node/lib/package/PackageInfo.ts

```diff
@@ -79,6 +79,7 @@ export class PackageInfo {
                     description: translatorsJson[moduleName].description,
                     moduleName: moduleName,
                     onboarding: onboardingReference,
+                    onboardingFlow: [],
                     onboardingProperties: onboardingProperties,
```

Comment:
This will need to be filled in, using the JSON data that was persisted in the package.json by LocalPackageSource.mergePackageInfo.